test/test_util.py

```diff
@@ -382,6 +382,44 @@ def test_dtype_from_element_str_and_bytes_dtypes(self) -> None:
             self.assertEqual(np.dtype(f'|S{size}'), dtype_from_element(bytes(size)))
             self.assertEqual(np.dtype(f'<U{size}'), dtype_from_element('x' * size))
 
+    def test_get_new_indexers_and_screen_a(self) -> None:
```

Comment:
I suggest more tests that call `get_new_indexers_and_screen()` with mixtures of positional args and kwargs and kwargs only.